wakuv2/common/message.go

```diff
@@ -17,12 +17,13 @@ import (
 )
 
 // MessageType represents where this message comes from
-type MessageType int
+type MessageType = string
 
 const (
-	RelayedMessageType MessageType = iota
-	StoreMessageType
-	SendMessageType
+	RelayedMessageType MessageType = "relay"
```

Comment:
Any specific reason for changing the type to string?

Reply:
The MessageType is currently being used only for adding more info to the log lines. I changed it to string so it's easier to read on the logs, otherwise they look like this:
```
DEBUG[08-05|09:55:27.969|github.com/status-im/status-go/wakuv2/waku.go:1535]                                                                 received new envelope                    messageType=0 envelopeHash=0x125cdf356292295dcb242cde779c401d5f08dbda5359e590afe9d961666c8616 contentTopic=/waku/1/0x2f380d40/rfc26 timestamp=1,722,851,727,870,663,494
```

Reply:
Actually, let me change the timestamp format too, because those decimal separators are annoying